### ISSUE TITLE
time is displayed but no value after clicking outside form

### ISSUE THREAD

**Issue by doyaaaaaken** [ankurk91/vue-flatpickr-component#196]
**I'm submitting a ...**  (check one with "x")
```
[x] Bug report
[ ] Feature request
[ ] Other, please describe
```

**Tell about your platform**
* flatPickr version : 4.6.6
* Vue.js version : 2.6.11
* Browser name and version : Chrome
* This package version : 8.1.5

**Minimal reproduction of the problem with instructions**

1. Access this JsFiddle page. https://jsfiddle.net/doyaaaaaken/Lor04ctu/2/
2. Click input box
3. Click outside input box
4. Then you can see time `12:00` , but there is no value on javascript. See below image.
![image](https://user-images.githubusercontent.com/5428401/91149826-8838f980-e6f6-11ea-9e6d-d9a7288f1573.png)

When I made flatpickr version down to 4.6.3, the bug was fixed.

**Comment by ankurk91** [comment]
I could not re-produce this issue.
I am using Latest MS Edge (Mac OS)

![screen-record](https://user-images.githubusercontent.com/6111524/91156309-a0eae480-e6e1-11ea-9bdb-86c58fb22a48.gif)

**Comment by doyaaaaaken** [comment]
@ankurk91 here it is.
![Aug-25-2020 20-05-23](https://user-images.githubusercontent.com/5428401/91167093-59c71880-e70e-11ea-8a53-eca7f55e25ff.gif)

**Comment by ankurk91** [comment]
ref

https://github.com/ankurk91/vue-flatpickr-component/pull/137
https://github.com/ankurk91/vue-flatpickr-component/issues/143


i will look into it

**Comment by ankurk91** [comment]
Should be fixed in v8.1.6
